x-pack/plugins/apm/server/lib/services/get_service_dependencies/get_metrics.ts

```diff
@@ -60,40 +135,17 @@ export const getMetrics = ({
               size: 100,
             },
             aggs: {
-              timeseries: {
-                date_histogram: {
-                  field: '@timestamp',
-                  fixed_interval: getBucketSize({ start, end, numBuckets })
-                    .intervalString,
-                  extended_bounds: {
-                    min: start,
-                    max: end,
-                  },
-                },
-                aggs: {
-                  latency_sum: {
-                    sum: {
-                      field: SPAN_DESTINATION_SERVICE_RESPONSE_TIME_SUM,
-                    },
-                  },
-                  count: {
-                    sum: {
-                      field: SPAN_DESTINATION_SERVICE_RESPONSE_TIME_COUNT,
-                    },
-                  },
-                  [EVENT_OUTCOME]: {
-                    terms: {
-                      field: EVENT_OUTCOME,
-                    },
-                    aggs: {
-                      count: {
-                        sum: {
-                          field: SPAN_DESTINATION_SERVICE_RESPONSE_TIME_COUNT,
-                        },
-                      },
-                    },
-                  },
-                },
+              current_period: {
+                filter: rangeQuery(start, end)[0],
```

Comment:
`rangeQuery(start, end)` and `rangeQuery(comparisonStart, comparisonEnd)` are called multiple times with the same args. There's some opportunity to consolidate these calls and reference them in multiple places.